src/services/completions.ts

```diff
@@ -198,7 +198,8 @@ namespace ts.Completions {
         });
     }
 
-    const completionNameForLiteral = JSON.stringify;
+    const completionNameForLiteral = (literal: string | number) =>
+        typeof literal as string === "bigint" ? literal + "n" : JSON.stringify(literal);

```

Comment:
same here, please add this to a list of changes after LKG.

Reply:
still not sure how this is supposed to work without `bigint` support from the underlying engine?

Reply:
I think the right thing to do here is to return a richer data structure than `number | string`. since `bigint` is not that common, i would make `literals` `number | string | { bigintValue: string }`